products/hashicorp-packer.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Hashicorp Packer
+category: server-app
+tags: hashicorp
+iconSlug: packer
+permalink: /hashicorp-packer
+alternate_urls:
+-   /packer
+releasePolicyLink: https://support.hashicorp.com/hc/en-us/articles/360021185113-Support-Period-and-End-of-Life-EOL-Policy
+changelogTemplate: https://github.com/hashicorp/packer/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: Standard Maintenance
+eoesColumn: Extended Maintenance
```

Comment:
I don't see documentation regarding an extended maintenance and I would remove this column:
```suggestion
```